### PR TITLE
upgrade uuid to v5

### PR DESCRIPTION
**Pull request by ranakan19**
upgrading to https://snyk.io/advisor/golang/github.com/gofrs/uuid/v5

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -218,6 +218,8 @@ github.com/goccy/go-json v0.10.2/go.mod h1:6MelG93GURQebXPDq3khkgXZkazVtN9CRI+MG
 github.com/godbus/dbus/v5 v5.0.4/go.mod h1:xhWf0FNVPg57R7Z0UbKHbJfkEywrmjJnf7w5xrFpKfA=
 github.com/gofrs/uuid v3.3.0+incompatible h1:8K4tyRfvU1CYPgJsveYFQMhpFd/wXNM7iK6rR7UHz84=
 github.com/gofrs/uuid v3.3.0+incompatible/go.mod h1:b2aQJv3Z4Fp6yNu3cdSllBxTCLRxnplIgP/c0N/04lM=
```
Comment (fbm3307) [inline go.sum:220]: The V3 version of gfrs should be removed i think ?

**Review by MatousJobanek — APPROVED** [review]

File: go.sum (hunk 1 above)
Comment (MatousJobanek) [reply go.sum:220]: try to run `go mod tidy`

**Comment by rajivnathan** [comment]
Is it worth switching to `github.com/google/uuid` to be consistent with https://github.com/kubesaw/ksctl/pull/34 ?

**Comment by ranakan19** [comment]
> Is it worth switching to `github.com/google/uuid` to be consistent with [kubesaw/ksctl#34](https://github.com/kubesaw/ksctl/pull/34) ?

when i created this PR, the intention was to use gofrs/uuid in ksctl as well. But then, I talked to @alexeykazakov and we chose to with google instead - so here is the new PR which replaces gofrs with google uuid - https://github.com/codeready-toolchain/toolchain-common/pull/395. 
I will mark this PR as closed now.

**Comment by ranakan19** [comment]
Closing this PR in favor of https://github.com/codeready-toolchain/toolchain-common/pull/395

**Pull request closed without merging**